### PR TITLE
[MBL-18067][Student] PDF annotation saving

### PR DESCRIPTION
**Pull request by domonkosadam**
Test plan: See ticket

refs: MBL-18067
affects: Student
release note: Local PDF annotations are now remain available after logout.

## Checklist

- [ ] Tested in dark mode
- [ ] Tested in light mode
- [ ] Approve from product

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

File: libs/login-api-2/src/main/java/com/instructure/loginapi/login/tasks/LogoutTask.kt [hunk 1]
```diff
@@ -105,7 +105,6 @@ abstract class LogoutTask(
                 CanvasRestAdapter.okHttpClient.cache?.evictAll()
                 RestBuilder.clearCacheDirectory()
                 Utils.getAttachmentsDirectory(ContextKeeper.appContext).deleteRecursively()
-                File(ContextKeeper.appContext.filesDir, "cache").deleteRecursively()
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply libs/login-api-2/src/main/java/com/instructure/loginapi/login/tasks/LogoutTask.kt:108]: Yeah, I think we can kepp this line and it's still gonna work fine

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by hermannakos — APPROVED** [review]
QA + 1

**Review by kristofnemere — APPROVED** [review]

**Pull request merged**